governance.qmd

```diff
@@ -0,0 +1,96 @@
+---
+title: "R-Multiverse Governance"
+---
+
+The R-Multiverse project is run by volunteers in a collaborative and open way. Portions of this governance policy have been inspired by that of the Apache Software Foundation.
```

Comment:
What does volunteers mean? I'm asking since it can be a confusing term. People might be volunteers from a project's perspective, but still get paid by something else.

Reply:
so maybe add another word? or what is the role of this information?

Reply:
Agree, I don't think 'volunteer' adds any particular significance here, so I'll re-word this.